productService/modules/getProductsList.ts

```diff
@@ -1,29 +1,37 @@
 import { APIGatewayProxyHandler, APIGatewayProxyEvent, Context } from 'aws-lambda';
-import products from '../mocks/products.json';
+import connectToDb from '../helpers/connectToDb';
 
 export const getProductsList: APIGatewayProxyHandler = async (event: APIGatewayProxyEvent, context: Context) => {
-  /* GET CAT PICTURE WITH AWAIT
-
-    import axios from 'axios'
-    const axios = require('axios');
-    const { data: { file } } = await axios.get('https://aws.random.cat/meow?ref=apilist.fun');
+  try {
+    console.log('method getProductsList was called without params');
+    const client = await connectToDb();
+    const { rows } = await client.query('select * from products inner join stocks on stocks.product_id = products.id');
```

Comment:
I suggest to join only `count` field from `stocks` table. Because in the result `id` and `product_id` will be the same values and `product_id` is redundant.